### PR TITLE
Player Movement

### PR DESCRIPTION
**Pull request by Belize22**
- Player can move left and right.
- Player is able to jump and free-falls to the ground while in midair. Jumping can only be done while on the ground.

### CODE REVIEW

File: Assets/CodeBase/Entities/player/player.cs [hunk 1]
```diff
@@ -4,15 +4,76 @@
 
 public class player : MonoBehaviour
```
Comment (Megan-VanHumbeck) [inline Assets/CodeBase/Entities/player/player.cs]: Class names/File names should be capitalized in C#

**Review by Megan-VanHumbeck — CHANGES_REQUESTED** [review]

**Comment by Megan-VanHumbeck** [comment]
The files are no longer loadable in unity because the file name does not match the class name. This isn't necessary for C# (though it is best practice) but it is necessary for Unity. 

`The associated script can not be loaded. Please fix any compile errors and assign a valid script`

**Review by Megan-VanHumbeck — APPROVED** [review]

File: Assets/CodeBase/Entities/player/Player.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    // Update is called once per frame
+    void Update()
+    {
+        inputProfile.checkInput();
+    }
+
+    private void OnCollisionEnter2D(Collision2D collision)
+    {
+        Debug.Log("Fuck");
```
Comment (JesseDes) [inline Assets/CodeBase/Entities/player/Player.cs]: Dont forget to clean logs we dont want the grader to see these LOL

**Review by JesseDes — APPROVED** [review]

File: Assets/CodeBase/Entities/player/Player.cs (hunk 2 above)
Comment (DavidoBod) [reply Assets/CodeBase/Entities/player/Player.cs]: hhahaha

**Review by DavidoBod — APPROVED** [review]

**Pull request merged**